.docs/README.md

```diff
@@ -229,22 +229,42 @@ public function detail(ApiRequest $request)
 }
 ```
 
-It converts request parameters to defined types. By default, you can use `int`, `float`, `string`. Or defined
-more types in neon.
+Available data types are `scalar`, `string`, `int`, `float`, `bool` and `datetime`.
+
+- `scalar`
+    - Tries to automatically convert value to boolean, int or float.
+    - Returns string, if conversion is not possible.
```

Comment:
@f3l1x I don't really like scalar. Conversion is too magical. And without conversion it would be just the same as string.

Reply:
We will see, we can drop it after all.